test/integration/integration_test_fixtures.py

```diff
@@ -178,6 +212,23 @@ def getTestServerRootUri(self):
     return super(HttpIntegrationTestBase, self).getTestServerRootUri(False)
 
 
+class MultiServerHttpIntegrationTestBase(IntegrationTestBase):
+  """
+  Base for running plain http tests against multiple Nighthawk test servers
+  """
+
+  def __init__(self, ip_version, backend_count):
```

Comment:
Can we add documentation for the constructor, including an Args section describing the arguments?

Reply:
Done (added some comments to the class, but not sure what to say for the constructor apart from Args)